filters/filters.go

```diff
@@ -108,9 +108,16 @@ type FilterContext interface {
 	Metrics() Metrics
 
 	// Allow filters to add Tags, Baggage to the trace or set the ComponentName.
+	//
+	// Deprecated: OpenTracing is deprecated, see https://github.com/zalando/skipper/issues/2104.
+	// Use opentracing.SpanFromContext(ctx.Request().Context()).Tracer() to get the proxy span Tracer.
 	Tracer() opentracing.Tracer
 
 	// Allow filters to create their own spans
+	//
+	// Deprecated: OpenTracing is deprecated, see https://github.com/zalando/skipper/issues/2104.
+	// Filter spans should be children of the proxy span,
+	// use opentracing.SpanFromContext(ctx.Request().Context()) to get the proxy span.
```

Comment:
This is not a proxy span but a "filters" span added by https://github.com/zalando/skipper/pull/890 if enabled https://github.com/zalando/skipper/blob/f6bf0336bafcaab3a7e71447d6925fdaf54d5c2a/proxy/tracing.go#L111-L119

Maybe we can change it to return ingress span.
Anyway it is not used by anything, e.g. `tracingTag` uses ingress span (which is `opentracing.SpanFromContext(ctx.Request().Context())`)

Reply:
See https://github.com/zalando/skipper/pull/3052